lib/engines/csvEngine.js

```diff
@@ -0,0 +1,163 @@
+var _ = require("lodash");
+var Bromise = require("bluebird");
+var converter = require("json-2-csv");
+var mqtt = require("mqtt");
+var fs = require('fs');
+
+var MQTT_PORT = "1883";
```

Comment:
FYI starting from NodeJS 4.x we can now use `const` keyword for constant variables and `let` for the others, `var` is kind of deprecated.

Reply:
Only for MQTT_PORT. Correct?